### ISSUE TITLE
A.02.01.iso.searching.parameters

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-discovery-service#34]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

As the list of SupportedISOQueryables and AdditionalQueryable are known an XPath could be present in the ATS.
'1.3 Metadata request' should be a prerequisite

**Comment by PeterParslow** [comment]
I'm sure I commented on this yesterday, proposing a solution. Actually, that was #36.
Proposed solution: reduce the purpose & test method of this test case to match the title, i.e. only to include the ISO parameters - i.e. to test only the first clause of Requirement 22. A.2.3 tests the other part, which details Requirement 5.
The XPath for this part is then (drawing on TG Example 7) ows:OperationsMetadata/ows:Operation[name="GetRecords"]/ows:Constraint[name="SupportedISOQueryables"]/ows:Value - that should return a set of nodes whose contents should match the middle column of TG Table 4, except for those rows in that table  where the middle column says "not supported" (which are then the subject of Tables 5 & 6, tested at A.2.3).
I'm not sure where the list of queryables in the current Test Method comes from. I've attempted an analysis in the attached, but can only assume I've missed some other source of information. Some of the unexpected ones (to me) may come from the draft SDS TG? I'm not sure that those are mandatory queryables on a Discovery Service.
[INSPIRE ISO queryables.docx](https://github.com/inspire-eu-validation/ats-discovery-service/files/206188/INSPIRE.ISO.queryables.docx)

Arguably, this test of the ISO queryables should be part of an OGC test, but I can't see any evidence at http://cite.opengeospatial.org/teamengine/about/csw/2.0.2/site/ that it is. That test may well concentrate on "generic" CSW, rather than the ISO AP.
